pytorch_lightning/strategies/ipu.py

```diff
@@ -62,6 +62,8 @@ def _move_float_tensors_to_half(self, batch: Any) -> Any:
 class IPUStrategy(ParallelStrategy):
     """Plugin for training on IPU devices."""
 
+    distributed_backend = "ipu_strategy"
```

Comment:
I don't think we require this. `ipu_strategy` doesn't imply anything about the training strategy being used.

Reply:
I think it does. If this wouldn't be the case, we wouldn't need a separate strategy for this, but a accelerator would have been sufficient

Reply:
Now the `distributed_backend` only means strategy_name. Every strategy should be registered and have a name; the other way around, strategies should have consistent user-facing behavior in terms of set up.  eg: `strategy=ddp` should equal to `strategy=DDPStrategy` and `strategy=ipu_strategy` should equal to `strategy=IPUStrategy`